scripts/macosx/build_qt5.sh

```diff
@@ -39,7 +39,7 @@ export QTDIR=$MIXXX_PREFIX/Qt-${VERSION_NUMBER}
 # Disable relocatable qt due to https://bugs.launchpad.net/mixxx/+bug/1871238
 # TODO(rryan): Disable framework build or use frameworks properly. Qt assumes
 # that if frameworks are enabled then they are used for deployment.
-./configure -opensource -prefix $QTDIR -sdk macosx${MIXXX_MACOSX_SDK} -system-sqlite -sql-sqlite -qt-zlib -platform macx-clang -release -confirm-license -securetransport -force-debug-info -no-feature-relocatable -nomake examples -nomake tests -skip qt3d -skip qtwebengine -I${MIXXX_PREFIX}/include -L${MIXXX_PREFIX}/lib
+./configure -opensource -prefix $QTDIR -sdk macosx${MIXXX_MACOSX_SDK} -ccache -system-sqlite -sql-sqlite -qt-zlib -platform macx-clang -release -confirm-license -securetransport -force-debug-info -nomake examples -nomake tests -skip qt3d -skip qtwebengine -I${MIXXX_PREFIX}/include -L${MIXXX_PREFIX}/lib
```

Comment:
Can you add a TODO that we switch it to CMake at some point? Then we can make sure that we use similar build options for windows/macos more easily.

Reply:
I don't think we should do that until Qt6 switches to CMake as its primary build system.

Reply:
ok